processor/metricstransformprocessor/README.md

```diff
@@ -0,0 +1,116 @@
+# Metrics Transform Processor <span style="color:red">**(UNDER DEVELOPMENT - NOT READY FOR USE)**</span>
```

Comment:
I don't think the html works, at least not when I look at the preview

Reply:
Thanks for catching this. For some reason, I didn't realize that at all because it works fine on my local preview.